modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/interval_selector_scheduler_event_link.js

```diff
@@ -0,0 +1,167 @@
+AUI.add(
+	'liferay-calendar-interval-selector-scheduler-event-link',
+	function(A) {
+		var AArray = A.Array;
+
+		var IntervalSelectorSchedulerEventLink = A.Component.create(
+			{
+				ATTRS: {
+					intervalSelector: {
+					},
+
+					schedulerEvent: {
+					}
+				},
+
+				AUGMENTS: [Liferay.PortletBase],
```

Comment:
Indeed, there is no need of that. I submitted a new PR without it, and with a minor fix as well. See https://github.com/brunobasto/liferay-portal/pull/666.

Thanks for your time!